src/include/duckdb/common/operator/cast_operators.hpp

```diff
@@ -205,4 +205,28 @@ template <> duckdb::dtime_t CastTimestampToTime::Operation(duckdb::timestamp_t i
 template <> duckdb::string_t CastFromTimestamp::Operation(duckdb::timestamp_t input, Vector &result);
 template <> duckdb::timestamp_t CastToTimestamp::Operation(string_t input);
 
+struct CastFromBlob {
+	template <class SRC> static inline string_t Operation(SRC input, Vector &result) {
+		throw duckdb::NotImplementedException("Cast from blob could not be performed!");
+	}
+
+	template <class SRC> static inline void ToHexString(SRC input, SRC &output) {
+		throw duckdb::NotImplementedException("Blob to hexadecimal string could not be performed!");
```

Comment:
Is there any reason this is templated when only the string_t -> string_t is implemented?

Reply:
You're right, there is no special reason to be templated, I will change it.